docker/go-control-plane/Dockerfile

```diff
@@ -0,0 +1,32 @@
+#[eostroukhov] TODO migrate to Alpine to decrease image size
```

Comment:
Another good reason is vulnerability scanning. It might be just a simple as adding this two lines and adding `as build` at line 2: https://github.com/grpc/grpc-go/blob/4a26a494081a67f29e8acf158913609c686e701d/interop/xds/client/Dockerfile#L32-L33

I recommend to give it a try, but it's fine to do it later in a follow-up.

Reply:
Did it (needed to learn more about the Go build)